packages/studio/src/components/common/renderIconForType.tsx

```diff
@@ -10,6 +10,7 @@ export default function renderIconForType(type: ElementType) {
     case ElementType.Components:
       return <Box />;
     case ElementType.Containers:
+    case ElementType.Layouts:
```

Comment:
Can we ask Product if there's an icon for Layouts? If there isn't, let's make sure they're cool with re-using the Container icon.

Reply:
I pinged Aaron!

Reply:
He said he's not sure, probably we can merge this in and update the icon later?

Reply:
Sounds good!